From: Johannes Thumshirn <jthumshirn@suse.de>
Date: Tue Apr 11 11:08:13 CEST 2017
Subject: Disable Werror building

Disbale the -Werror compiler flag due to a new warning in GCC 7. Although the
warning is legitim it is a false positive in this place.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -5,7 +5,7 @@ sbin_PROGRAMS = fcoeadm fcoemon fcping f
 AM_CPPFLAGS = -I${srcdir}/include -I${builddir}/include
 ## pass the sysconfdir into the C proprocessor
 AM_CPPFLAGS += -DSYSCONFDIR="\"${sysconfdir}\"" -D_FORTIFY_SOURCE=2
-AM_CFLAGS = -Wall -Wformat=2 -Werror -Wmissing-prototypes -Wstrict-prototypes
+AM_CFLAGS = -Wall -Wformat=2 -Wmissing-prototypes -Wstrict-prototypes
 
 ## rules for building fcoeadm
 fcoeadm_SOURCES = fcoeadm.c fcoeadm_display.c
